package.json

```diff
@@ -30,7 +30,7 @@
     "@guardian/tsconfig": "^0.2.0",
     "@types/jest": "^27.4.1",
     "@types/js-yaml": "^4.0.5",
-    "@types/node": "^20.3.3",
+    "@types/node": "^18.17.0",
```

Comment:
FYI, our [recommendation for JavaScript is to use exact package versions](https://github.com/guardian/recommendations/blob/main/dependencies.md#javascript).